app/views/whois_records/_legal_person_for_authorized.html.erb

```diff
@@ -0,0 +1,57 @@
+Domain:
```

Comment:
There is a duplication between `legal_person_for_authorized.json.jbuilder` and `legal_person.json.jbuilder`. I see a clear pattern of undisclosed data ("Not Disclosed - Visit www.internet.ee for webbased WHOIS"). 

Perhaps we could unify those templates?

Reply:
I decided against it since you end up with an if block, something I find rather unreadable in templates. I'd rather live with the repetition.

```ruby
if @verified
  json.array!(whois_record.json['tech_contacts']) do |_contact|
    json.name 'Not Disclosed'
    json.email 'Not Disclosed'
    json.changed 'Not Disclosed'
  end
else
  json.array!(whois_record.json['admin_contacts']) do |_contact|
    json.name 'Not Disclosed - Visit www.internet.ee for webbased WHOIS'
    json.email 'Not Disclosed - Visit www.internet.ee for webbased WHOIS'
    json.changed 'Not Disclosed - Visit www.internet.ee for webbased WHOIS'
  end
```

I prefer having multiple templates to one that covers multiple, hence the design where `WhoisRecord` instance tells the controller which template to render, not the controller/template asking the `WhoisRecord` about its properties.